types/twit/index.d.ts

```diff
@@ -314,6 +314,13 @@ declare module 'twit' {
             name?: string;
             description?: string;
             mode?: 'public' | 'private';
+            exclude_reply_user_ids?: string[];
```

Comment:
This could probably be a string if there is only one ID.

```suggestion
            exclude_reply_user_ids?: string | string[];
```